docs/gc.md

```diff
@@ -132,15 +132,43 @@ There are a number of advantages to using arraylets.
 
 ### `metronome` policy
 
-[`-Xgcpolicy:metronome`](xgcpolicy.md#metronome-aix-linux-x86-only) is designed for applications that require precise response times. Garbage collection occurs in small interruptible steps to avoid stop-the-world pauses. This policy is available only on x86 Linux and AIX platforms.
+**(Linux on x86-64 and AIX platforms only)**
 
-### `optavgpause` policy
+[`-Xgcpolicy:metronome`](xgcpolicy.md#metronome-aix-linux-x86-only) is designed for applications that require a precise upper bound on collection pause times as well as specified application utilization: the proportion of time that the application is permitted to use, with the remainder being devoted to GC. The realtime GC runs in short interruptible bursts to avoid stop-the-world pauses.
```

Comment:
Those interruptible bursts are also a series of STW pauses, but very short. So either we should say 'to avoid _long_
STW pauses', or we should complete rephrase things like: 'realtime breaks done a single GC STW cycle into a burst/series of very short STW increments'

Reply:
Just added 'long', as suggested.